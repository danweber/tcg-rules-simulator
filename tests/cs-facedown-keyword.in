VERSION:2
FIELD1:Sample,DW1-Van,FACEDOWN,CS1-Garbage,CS1-Paper,CS2-09,FACEDOWN,CS1-Tiny,FACEDOWN,CS1-17
GAME:P1 T3 MAIN 6
DUMP:KEYWORDS

