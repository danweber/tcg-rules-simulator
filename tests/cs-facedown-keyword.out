FIELD1:Sample,DW1-Vanillamon,FACEDOWN,CS1-Garbagecanlid,CS1-Papershield,CS2-Masamune,FACEDOWN,CS1-Tinyshield,FACEDOWN,CS1-Gideontheguardian,SECURITY ATTACK [-+]\D
FIELD2:
GAME:P1 T3 MAIN 6
